gr-iio/grc/iio.tree.yml

```diff
@@ -7,11 +7,6 @@
     - iio_attr_sink
     - iio_attr_updater
     - iio_dds_control
-  - 'FMComms':
-    - iio_fmcomms2_source
-    - iio_fmcomms2_sink
-    - iio_fmcomms5_source
-    - iio_fmcomms5_sink
```

Comment:
Is there a reason why you're dropping `fmcomms[25]`?

Reply:
Namely it's to get the pluto support out there with the updated API, then add these back later as people who have the means to test them can try it out.